api/txspool_test.go

```diff
@@ -193,14 +193,6 @@ func TestPoolTxs(t *testing.T) {
 	jsonTxReader = bytes.NewReader(jsonTxBytes)
 	err = doBadReq("POST", endpoint, jsonTxReader, 400)
 	require.NoError(t, err)
-	// Wrong rq
-	badTx = tc.poolTxsToSend[0]
-	badTx.RqFromIdx = 30
-	jsonTxBytes, err = json.Marshal(badTx)
-	require.NoError(t, err)
-	jsonTxReader = bytes.NewReader(jsonTxBytes)
-	err = doBadReq("POST", endpoint, jsonTxReader, 409)
-	require.NoError(t, err)
```

Comment:
Deleted this test, bcs internally we didn't check the correctness of the RqFromIdx. This test worked before changes, bcs 409 is the code for ItemAlreadyExist. Now we can update txs in the pool, so this test no longer working.